src/pudl/transform/ferc1.py

```diff
@@ -2801,6 +2802,40 @@ def process_instant_xbrl(self, df: pd.DataFrame) -> pd.DataFrame:
         return df.reset_index()
 
 
+class DepreciationAmortizationSummaryFerc1TableTransformer(
+    Ferc1AbstractTableTransformer
+):
+    """Transformer class for :ref:`depreciation_amortization_summary_ferc1` table."""
+
+    table_id: TableIdFerc1 = TableIdFerc1.DEPRECIATION_AMORTIZATION_SUMMARY_FERC1
+    has_unique_record_ids: bool = False
+
+    def map_ferc_accounts(self, df: pd.DataFrame) -> pd.DataFrame:
```

Comment:
In the `plant_in_service_ferc1` table, we did some manual metadata annotation in the `merge_xbrl_metadata()` method, and I wonder if we should just use that method across the board for any work of this kind?  Maybe it should have a more expansive name that can encompass purely manual annotations and allow a parametrization that doesn't try to do the programmatic merge? @cmgosnell what do you think the right place is for this kind of manual metadata munging?

Reply:
After chatting with Christina I updated this to use the same pattern as the tables insofar as it's possible. So I renamed this `merge_xbrl_metadata` and used the same parameters & function to do the merge, but with the small hand-compiled metadata dataframe, rather than the one from the XBRL Taxonomy.